analysed-packages/libxcrypt/version-4.4.33/libxcrypt-4.4.33-OSS-disclosure.txt

```diff
@@ -0,0 +1,2572 @@
+liability in connection with the Program, unless a warranty or assumption of
+liability accompanies a copy of the Program in return for a fee.
+
+END OF TERMS AND CONDITIONS
+
+How to Apply These Terms to Your New Programs
+
+If you develop a new program, and you want it to be of the greatest possible use
+to the public, the best way to achieve this is to make it free software which
+everyone can redistribute and change under these terms.
+
+To do so, attach the following notices to the program. It is safest to attach
+them to the start of each source file to most effectively state the exclusion of
+warranty; and each file should have at least the "copyright" line and a pointer
+to where the full notice is found.
+
+<one line to give the program's name and a brief idea of what it does.>
+
+Copyright (C) <year> <name of author>
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later
+version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+If the program does terminal interaction, make it output a short notice like
+this when it starts in an interactive mode:
+
+<program> Copyright (C) <year> <name of author>
+
+This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
+
+This is free software, and you are welcome to redistribute it under certain
+conditions; type `show c' for details.
+
+The hypothetical commands `show w' and `show c' should show the appropriate
+parts of the General Public License. Of course, your program's commands might be
+different; for a GUI interface, you would use an "about box".
+
+You should also get your employer (if you work as a programmer) or school, if
+any, to sign a "copyright disclaimer" for the program, if necessary. For more
+information on this, and how to apply and follow the GNU GPL, see
+<https://www.gnu.org/licenses/>.
+
+The GNU General Public License does not permit incorporating your program into
+proprietary programs. If your program is a subroutine library, you may consider
+it more useful to permit linking proprietary applications with the library. If
+this is what you want to do, use the GNU Lesser General Public License instead
+of this License. But first, please read <https: //www.gnu.org/licenses/why-not-
+lgpl.html>.
+
+License text of the Autoconf-exception-generic-3.0 as an exception to the
+GPL-3.0-or-later:
+
+As a special exception to the GNU General Public License, if you distribute this
```

Comment:
This is not the correct text of the exception, in the files m4/zw_automodern.m4 and m4/zw_simple_warnings.m4 it is stated: 
Under Section 7 of GPL version 3, you are granted additional
permissions described in the Autoconf Configure Script Exception,
version 3.0, as published by the Free Software Foundation.

Because only two files in this source tree are released
under GPLv3 with exceptions, neither the GPLv3 nor the exception are
distributed with this source tree. Copies can be retrieved from
https://www.gnu.org/licenses/

The text of the Autoconf Configure Script Exception, version 3.0 is https://www.gnu.org/licenses/autoconf-exception-3.0.en.html:

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

This Exception is an additional permission under section 7 of the GNU General Public License, version 3 ("GPLv3"). It applies to a given file that bears a notice placed by the copyright holder of the file stating that the file is governed by GPLv3 along with this Exception.

The purpose of this Exception is to allow distribution of Autoconf's typical output under terms of the recipient's choice (including proprietary).

0. Definitions.
"Covered Code" is the source or object code of a version of Autoconf that is a covered work under this License.

"Normally Copied Code" for a version of Autoconf means all parts of its Covered Code which that version can copy from its code (i.e., not from its input file) into its minimally verbose, non-debugging and non-tracing output.

"Ineligible Code" is Covered Code that is not Normally Copied Code.

1. Grant of Additional Permission.
You have permission to propagate output of Autoconf, even if such propagation would otherwise violate the terms of GPLv3. However, if by modifying Autoconf you cause any Ineligible Code of the version you received to become Normally Copied Code of your modified version, then you void this Exception for the resulting covered work. If you convey that resulting covered work, you must remove this Exception in accordance with the second paragraph of Section 7 of GPLv3.

2. No Weakening of Autoconf Copyleft.
The availability of this Exception does not imply any general presumption that third-party software is unaffected by the copyleft requirements of the license of Autoconf.

Please correct this. Additionally please provide the text of the below given sections as license comment in the SPDX files:
The file contains the GPL-3.0-or-later boilerpate and the follwing text:
Under Section 7 of GPL version 3, you are granted additional
permissions described in the Autoconf Configure Script Exception,
version 3.0, as published by the Free Software Foundation.
dnl
Because only two files in this source tree are released
under GPLv3 with exceptions, neither the GPLv3 nor the exception are
distributed with this source tree. Copies can be retrieved from
https://www.gnu.org/licenses/
we copied the original text of the Autoconf Configure Script Exception, version 3.0, from https://www.gnu.org/licenses/autoconf-exception-3.0.en.html: and concluded GPL-3.0-or-later WITH Autoconf-exception-3.0 as license of this file.
The information was retrieved on 8th of Sept 2024